### DISCUSSION TITLE
Using Haxeflixel with npm

### DISCUSSION THREAD

**Discussion by rishavs** [HaxeFlixel/flixel#3209] [General]
Hi

Given that openfl can be installed via npm, allowing the user of typescript and getting things like live reload for free, is this something we can also do for flixel? If it is possible, can someone share the steps for this? If not, can this be a feature request for the library?

**Comment by moxie-coder** [comment]
to simply put it, no you cannot, HaxeFlixel and OpenFL are very two different libraries for Haxe, while Flixel is powered by openfl, it wouldn’t be quite possible to get it on npm

**Reply by Geokureli** [reply]
yeah, as far as I know, none of the current contributors to HaxeFlixel are using, or perhaps, familiar with npm, so adding support for it would need to be the effort of someone who is

**Comment by rishavs** [comment]
Thanks.
